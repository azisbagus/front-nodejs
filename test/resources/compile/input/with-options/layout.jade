!!!
html
    head
        title Layout
        link(rel='stylesheet', href='bootstrap/css/bootstrap.css')
        link(rel='stylesheet', href='style.css')

    body
        include index
