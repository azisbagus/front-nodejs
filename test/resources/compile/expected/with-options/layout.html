<!DOCTYPE html>
<html>
  <head>
    <title>Layout</title>
    <link rel="stylesheet" href="bootstrap/css/bootstrap.css">
    <link rel="stylesheet" href="style.css">
  </head>
  <body>
    <div class="container">
      <header>
        <h1>Steer the Beat</h1>
      </header>
      <footer>
        <p>Copyright &copy; 2012 Steer the Beat.</p>
      </footer>
    </div>
  </body>
</html>